### ISSUE TITLE
Multiple modules/files for custom executor

### ISSUE THREAD

**Issue by janandreschweiger** [jina-ai/jina#1546]
Hey everyone,

we have build a very complicated executor and want to split the source code into different files/modules. I would expect that it works like for the [MongoDBIndexer](https://github.com/jina-ai/jina-hub/blob/1abd7f54b4c3d5dff81375f9eb54d0dae8266f21/indexers/keyvalue/MongoDBIndexer/__init__.py#L29).

I tried various ways, but I always get one of the following two errors:
- ImportError('attempted relative import with no known parent package')
- ModuleNotFoundError("No module named 'segment'")

It would be cool if someone could have a look on this. Thank you!

**Comment by JoanFM** [comment]
> Hey everyone,
> 
> we have build a very complicated executor and want to split the source code into different files/modules. I would expect that it works like for the [MongoDBIndexer](https://github.com/jina-ai/jina-hub/blob/1abd7f54b4c3d5dff81375f9eb54d0dae8266f21/indexers/keyvalue/MongoDBIndexer/__init__.py#L29).
> 
> I tried various ways, but I always get one of the following two errors:
> 
> * ImportError('attempted relative import with no known parent package')
> * ModuleNotFoundError("No module named 'segment'")
> 
> It would be cool if someone could have a look on this. Thank you!

Hey @janandreschweiger ,

thanks for opening the issue, could you share some sample code or structure that can help us take a look at it?

**Comment by janandreschweiger** [comment]
Sure! Thanks @JoanFM for your help.

crafter.yml
```
!CustomCrafter
metas:
  py_modules: custom_crafter.py
```

custom_crafter.py
```
class CustomCrafter(BaseSegmenter):
    def craft(self, buffer: bytes, *args, **kwargs):
        from helper import helper_function
        # additional code
```

helper.py
```
def helper_function():
    pass
```

Structure:
flows/
  search-flow.yml
pods/
  crafter.yml
  helper.py
  custom_crafter.py

**Comment by bhavsarpratik** [comment]
Hey @janandreschweiger, thanks for trying Jina! 
You can have a look at this [example](https://github.com/jina-ai/examples/blob/master/multimodal-search-tirg/TirgImageEncoder/__init__.py) on how we use a custom executor with extra helper files. You might be missing `sys.path.append(".")`

**Comment by hanxiao** [comment]
@janandreschweiger 

not sure if you know it but `py_modules` can be a list, say if A depends on B, which depends on C then 
```yaml
py_modules:
  - C.py
  - B.py
  - A.py
```

**Comment by janandreschweiger** [comment]
Thanks for your kind help @hanxiao @bhavsarpratik!
I did it exactly like in your [example](https://github.com/jina-ai/examples/blob/master/multimodal-search-tirg/TirgImageEncoder/config.py). I tried it again today and it still doesn't work.

Do you have a unit test that ensures that this error isn't caused by the latest version?
If not, I could provide you with a minimal working example.

**Comment by ace-kay-law-neo** [comment]
I get the same error. Do you think that it is caused by jina @bhavsarpratik?

**Comment by hanxiao** [comment]
@janandreschweiger @ace-kay-law-neo Problem solved and rolled out in `0.9.2`, `py_modules` usage can be found in https://github.com/jina-ai/jina-hub/blob/master/README.md#use-py_modules-to-import-multiple-files

The test corresponds to @janandreschweiger your case can be found in https://github.com/jina-ai/jina/blob/master/tests/integration/issues/github_1546/test_pymodules_import.py#L73

**Comment by janandreschweiger** [comment]
Thank you @hanxiao. You are the best.
`from jinahub.helper import helper_function` did the trick.
